stats/lib/platform-stats-fetchers.js

```diff
@@ -13,25 +8,27 @@ import {
  * @param {import('./typings.js').DateRangeFilter} filter
  */
 export const fetchDailyStationCount = async (pgPool, filter) => {
-  return await getDailyDistinctCount({
-    pgPool,
-    table: 'daily_stations',
-    column: 'station_id',
-    filter
-  })
+  const { rows } = await pgPool.query(`
+    SELECT day::TEXT, station_count
+    FROM daily_measurements_summary
+    WHERE day >= $1 AND day <= $2
+    ORDER BY day
+  `, [filter.from, filter.to])
+  return rows
 }
 
 /**
  * @param {Queryable} pgPool
  * @param {import('./typings.js').DateRangeFilter} filter
  */
 export const fetchMonthlyStationCount = async (pgPool, filter) => {
-  return await getMonthlyDistinctCount({
```

Comment:
These functions are now only used once each: https://github.com/filecoin-station/spark-stats/blob/a6455da4c58810a3e32b83adbaaef7dd5ba8ecdd/stats/lib/stats-fetchers.js#L67-L85

Would you prefer to remove these function definitions and place the logic directly in their calling function?

Reply:
It's great that you pointed this out, I would not realise that these helpers are used in a single place.

I am fine either way.

Since we already created these helpers, there is little (if any) maintenance overhead, and we may have a use for them in the future, I slightly prefer to keep them around.

@juliangruber thoughts?

Reply:
Since the outer functions only call the helper and nothing else, I would prefer to inline them, to reduce tech debt (the downside of the current solution is it takes you an unnecessary extra step to get to the business logic)

Reply:
in that case, let's do what @juliangruber wrote